libs/agno/tests/unit/memory/test_memory_manager_async.py

```diff
@@ -1,11 +1,14 @@
 from types import MethodType
-from typing import Dict, List, Optional, Tuple
+from typing import TYPE_CHECKING, Dict, List, Optional, Tuple
 
 import pytest
 
 from agno.db.base import AsyncBaseDb
 from agno.memory import MemoryManager, UserMemory
 
+if TYPE_CHECKING:
+    from agno.tracing.schemas import Span, Trace
+
 
 class DummyAsyncMemoryDb(AsyncBaseDb):
```

Comment:
This is a very weird place to define this... We should move this to a higher up file